libretro-common/audio/conversion/float_to_s16.c

```diff
@@ -97,22 +97,25 @@ void convert_float_to_s16(int16_t *out,
    size_t i          = 0;
```

Comment:
I initially thought I might need to modify this function to support mono input, so I commented it to understand what it was doing. But it turns out that mono audio works fine with this function anyway.